Content.Server/Silicons/Laws/SiliconLawSystem.cs

```diff
@@ -71,6 +71,22 @@ private void OnMindAdded(EntityUid uid, SiliconLawBoundComponent component, Mind
             actor.PlayerSession.Channel, colorOverride: Color.FromHex("#2ed2fd"));
     }
 
+    private void OnLawProviderMindAdded(EntityUid uid, SiliconLawProviderComponent component, MindAddedMessage args)
```

Comment:
```suggestion
    private void OnLawProviderMindAdded(Entity<SiliconLawProviderComponent> ent, ref MindAddedMessage args)
```
new code convention, we always pair up the entity and component